modules/tests/test/vm/operations.go

```diff
@@ -7,12 +7,14 @@ import (
 	"github.com/kubevirt/kubevirt-tekton-tasks/modules/tests/test/dataobject"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
 	kubevirtv1 "kubevirt.io/api/core/v1"
 	kubevirtcliv1 "kubevirt.io/client-go/kubecli"
 	cdicliv1beta1 "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned/typed/core/v1beta1"
 )
 
-func WaitForVM(kubevirtClient kubevirtcliv1.KubevirtClient,
+func WaitForVM(k8sClient *kubernetes.Clientset,
```

Comment:
Can't the `kubecli.KubevirtClient` be used for everything? It implements `kubernetes.Clientset` and provides the CDI client with method `CdiClient()`.